chirpus-2/index.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto&display=swap');
+body {
+    margin: 10px;
+    background-color: #f9fafc;
+    font-family: 'Open-Sans', sans-serif;
+}
+.nav {
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    color: #69696a;
+    padding: 20px;
+}
+.nav__logo {
+    position: absolute;
+    padding: 5px;
+    left: 10px;
+    color: #000;
+}
+.nav__links__link {
+    display: flex;
+    justify-content: flex-end;
+    padding: 5px;
+}
+.button {
+    color: #fff;
+    background-color: #000;
+    border-radius: 25px;
+    padding: 5px;
+    width: 100%;
+    height: 30px;
+}
+.wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+.content {
+    display: flex;
+    flex-direction: column;
+}
+.content h1 {
+    color: #000;
+}
+.content p {
+    color: #69696a;
+    border-left: 2px solid #69696a;
+    padding: 0 25px;
+    font-size: 14px;
+    margin: auto;
+}
+.graphic {
+    position: relative;
+    display: none;
+}
+.graphic__circle {
+    width: 75px;
+    height: 75px;
+    background-color: rgba(207, 130, 213, 0.6);
+    border-radius: 50%;
+    position: absolute;
+    top: -100px;
+    right: 0px;
+}
+.graphic__circle--large {
+    width: 150px;
+    height: 150px;
+    background-color: rgba(247, 136, 167, 0.6);
+    top: -150px;
+    right: 50px;
+}
+.graphic__circle--medium {
+    width: 100px;
+    height: 100px;
+    background-color: rgba(247, 193, 162, 0.7);
+    top: -50px;
+    right: 25px;
+}
+.footer {
+    background-color: rgba(247, 193, 162, 0.7);
+    display: flex;
+    position: fixed;
+    bottom: 0px;
+    left: 0px;
+    width: 100%;
+    height: 20px;
+}
+.footer__logo {
+    position: absolute;
+    top: 0px;
+    left: 10px;
+    color: #000;
+}
+.footer__links {
+    position: absolute;
+    top: 0px;
+    right: 10px;
+    color: #69696a;
+    text-decoration: none;
+}
+.footer__links__link {
+    padding-left: 10px;
+}
+
+@media (min-width: 768px) {
+    .nav {
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-end;
+    }
+    .content {
+        width: 50%;
+    }
+    .nav__button {
+        width: 100px;
+    }
+    .graphic {
+        display: block;
+        width: 100%;
+    }
+}
```

Comment:
Inside your media query is where you could target your nav__links class. If you gave that class display: flex your links would lay next to each other in a row instead of stacked in a column. Row is the default layout when we add display: flex.